server/database/queries/selectProjectDetails.js

```diff
@@ -1,7 +1,7 @@
 const connect = require('./../config/connection');
 
 const selectProjectDetails = projectID => connect.query(
-  'SELECT projects.*, user_id, full_name FROM  users_projects INNER JOIN projects ON projects.id = users_projects.project_id INNER JOIN users on users.id = user_id WHERE project_id = $1;',
+  'SELECT projects.*, users.id as user_id, users.full_name FROM users_projects INNER JOIN projects ON projects.id = users_projects.project_id INNER JOIN users on users.id = user_id WHERE project_id = $1;',
```

Comment:
this is the change we made to the ```selectProjectDetails``` query @aminalakhsham

Reply:
So long line :man_facepalming: